bundler/src/generate_docs.ts

```diff
@@ -81,220 +111,59 @@ For instructions on how to write constraint templates, see [How to write your ow
 In addition to browsing all [Available Templates](#available-templates) and [Sample Constraints](#sample-constraints),
 you can explore these policy bundles:
 
+- [CFT Scorecard](./bundles/scorecard-v1.md)
 - [CIS v1.0](./bundles/cis-v1.0.md)
 - [CIS v1.1](./bundles/cis-v1.1.md)
+- [Forseti Security](./bundles/forseti-security.md)
 - [GKE Hardening](./bundles/gke-hardening-v2019.11.11.md)
-- [CFT Scorecard](./bundles/scorecard-v1.md)
 
 ## Available Templates
 
-${mdTable(templates)}
+${MD_TABLE(templates)}
 
 ## Sample Constraints
 
 The repo also contains a number of sample constraints:
 
-${mdTable(samples)}
+${MD_TABLE(samples)}
 `;
 
   const templateDocPath = path.join(sinkDir, "index.md");
   fileWriter.write(templateDocPath, templateDoc);
+}
 
-  // Document bundles
-  library.bundles.forEach((bundle) => {
+function generateBundleDocs(
+  bundleDir: string,
+  fileWriter: FileWriter,
+  library: PolicyLibrary
+) {
+  library.bundles.forEach(bundle => {
     const constraints = [["Constraint", "Control", "Description"]];
-    bundle.getConfigs().forEach((o) => {
-      const name = `[${getName(o)}](${getPath(o, "../../")})`;
-      const control = bundle.getControl(o);
-      const description = getDescription(o);
-    
-      constraints.push([name, control, description]);
-    });
+    bundle
+      .getConfigs()
+      .sort((a, b) => PolicyConfig.compare(a, b))
```

Comment:
nit. Couldn't this just be:
```suggestion
      .sort(PolicyConfig.compare)
```

Reply:
Ah yes, nice! Much nicer..